config/default.json

```diff
@@ -1,14 +1,14 @@
 {
   "accounts_url": "https://husky.codechain.io/explorer/api/accounts" ,
+  "oil_utxo_url": null,
```

Comment:
What the `oil_utxo_url` means is unclear.

IMO, adding the indexer URL and the oil's asset type to the config file is more intuitive.

Reply:
Fixed